engines/e_cng.c

```diff
@@ -0,0 +1,912 @@
+/*
+ * Copyright 2017 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#ifdef _WIN32
+# include <windows.h>
+# include <wincrypt.h>
```

Comment:
> I have verified that wincrypt.h includes ncrypt.h indeed.

Question was if *all* of them do. But of course one can always say that if wincrypt.h doesn't include ncrypt.h, then *we* don't compile it. It would be appropriate to mention this in more verbose comment...